recipes/tinyalsa/all/conanfile.py

```diff
@@ -2,13 +2,16 @@
 from conans.errors import ConanInvalidConfiguration
 import os
 
+required_conan_version = ">=1.33.0"
+
 class TinyAlsaConan(ConanFile):
     name = "tinyalsa"
     license = "BSD-3-Clause"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/tinyalsa/tinyalsa"
-    topics = ("conan", "tiny", "alsa", "sound", "audio", "tinyalsa")
+    topics = ("tiny", "alsa", "sound", "audio", "tinyalsa")
     description = "A small library to interface with ALSA in the Linux kernel"
+    exports_sources = ["patches/*",]
```

Comment:
Consider exporting only the patches needed. 
https://github.com/conan-io/conan-center-index/issues/7790

Reply:
@ericriff 
Thank you for review.
I know your point.

This discussion seems to be not finished. 
When it is finished, I am going to follow the result.